main.js

```diff
@@ -43,6 +43,73 @@ for (let i = 0; i < ex10.length; i++) {
         console.log(ex10[i]);}
 }
 
+//ДЗ 3 задание 5
+function isPalindrome(str) {
+    let strLen = str.length,
+        strReverse = str.split('').reverse().join('');
+    if (strReverse == str) {
+        return true;
+    } else {
+        return false;
+    }
+}
+test = isPalindrome('шалаш');
+test2 = isPalindrome('даша');
+console.log(test);
+console.log(test2);
```

Comment:
use let or const to initialize variables (or if this variables were used at previous tasks - init new variables to separate logic)